guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -29,10 +29,17 @@ Any Liquidity providers (LPs) can acquire JLP by swapping for it on Jupiter Swap
 
 JLP can also be sold via Jupiter Swap. Simply swap JLP for your desired asset. The JLP token could either be transferred to another trader or may be redeemed by the JLP pool, burning the JLP token, and releasing some of the currency contained in the pool.
 
-:::tip
-The best way to purchase or exit JLP is always via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
-:::
+**Purchasing/Exiting JLP** 
+
+The best way to purchase or exit JLP is via [Jupiter Swap](https://jup.ag/swap/USDC-JLP).
+
+When using Jupiter Swap or programs that integrate Jupiter Swap, the minting and burning of JLP tokens occur automatically. When you swap in and out of JLP, Jupiter Swap will find the best route to mint/burn JLP automatically, or swap through other DEXes if the route offered is better.
+
+:::info
+This automatic minting/burning mechanism is unique to Jupiter Swap and programs that route via Jupiter Swap. If you're interacting directly on a different DEX, you will trade JLP at the price offered by the DEX instead of the virtual price of JLP.
 
+These transactions would also not affect the overall JLP supply - only the Jupiter Swap program has the authority to mint and burn JLP tokens.
```

Comment:
jupiter swap program has no authority to mint or burn, Jupiter perp program has it and swap integrated jupiter perp program

Reply:
Yeah I think good idea to mention the actual program with authority. Before this I put Jupiter Swap because to generalize it, but it's wrong for sure; updated it here: https://github.com/jup-ag/space-station/pull/499/commits/34d3ebc647b906822e3ca0ce5cd9fe82d4a3f643

WDYT?